test/cypress/integration/admin/admin-helpers.cy.js

```diff
@@ -142,6 +142,7 @@ export class AdminPage {
   clickBlock(name) {
     return cy
       .get(`.action-add-block-${name}`, { timeout: 60000 })
+      .scrollIntoView()
```

Comment:
QQ here. I noticed in [the docs](https://docs.cypress.io/api/commands/scrollintoview) it says:

> It is [unsafe](https://docs.cypress.io/guides/core-concepts/retry-ability#Only-queries-are-retried) to chain further commands that rely on the subject after .scrollIntoView().

Should this perhaps instead be something like?

```js
cy.get(`.action-add-block-${name}`, { timeout: 60000 })
    .scrollIntoView()
    .should('be.visible');
return cy.get(`.action-add-block-${name}`, { timeout: 60000 }).click();
```

Reply:
Thanks @anselmbradford, nice find. Hopefully addressed satisfactorily in 7a140d0.